scipy/ndimage/__init__.py

```diff
@@ -86,6 +86,7 @@
    minimum_position
    standard_deviation - Standard deviation of an N-D image array
    sum - Sum of the values of the array
```

Comment:
can you remove `sum` from this list?

Reply:
we typically don't keep aliases in these lists, so it's clear which function to prefer

Reply:
Done.